### DISCUSSION TITLE
Do codemagic support WASM, and how do I configure it when my codebase is in Flutter ?

### DISCUSSION THREAD

**Discussion by RoarGronmo** [codemagic-ci-cd/codemagic-docs#2560] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

-

### Steps to reproduce

Can't find any information on how to configure codemagic to support wasm, such as:

1. Setting theese configiration parameters:

`Cross-Origin-Embedder-Policy` with value `credentialless` or `require-corp`
`Cross-Origin-Opener-Policy` with value `same-origin`

Please check flutter documentation and give us a method do comply.
https://docs.flutter.dev/platform-integration/web/wasm

### Expected results

information

### Actual results

no information

**Comment by himesh-cm** [comment]
Hey @RoarGronmo 

I hope this message finds you well. I just wanted to let you know that I have been looking into the Flutter documentation and Flutter does indeed support web assembly.
​
You can find all the details in the documentation here: https://docs.flutter.dev/platform-integration/web/wasm.

I also wanted to mention that the final code that Flutter compiles is compatible with all browsers, so for testing purposes, you can use Codemagic static pages. However, please keep in mind that these pages are intended for testing purposes only. For production use, I would recommend considering other platforms such as Cloudflare Pages or Firebase Hosting to host your Flutter web apps.

I hope this information is helpful to you. Please let me know if you have any further questions or need any assistance. Have a great day!

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
